extensions/amp-list/0.1/amp-list.js

```diff
@@ -455,7 +455,7 @@ export class AmpList extends AMP.BaseElement {
     // In the load-more case, we allow the container to be height auto
     // in order to reasonably make space for the load-more button and
     // load-more related UI elements underneath.
-    if (!this.loadMoreEnabled_) {
+    if (!this.loadMoreEnabled_ && !this.isLayoutContainer_) {
```

Comment:
There's already a comment explaining the first clause, let's explain the second too.